### ISSUE TITLE
Online training

### ISSUE THREAD

**Issue by jwallwork23** [Cambridge-ICCS/FTorch#111]
The ability to do online training in FTorch would be really useful. Presumably achieving this would mainly involve wrapping other parts of the (Py)Torch API.

@TomMelt suggested that this piece of development work plus an application code use case could work nicely as a summer placement or internship project.

**Comment by jatkinson1000** [comment]
Yes, a discussion was started in #22 and I think this would be a significant step for us, and very useful.

I did suggest this to Paul previously and was, but he had reasons not to take a summer student or placement.
However, that has now changed, so if @TomMelt and @MarionBWeinzierl are in support we can perhaps revisit?
I spoke to the directors who were in support at the time (we are having science and comms interns again), but I don't know if the funding window has now passed...

**Comment by jwallwork23** [comment]
Ah okay thanks @jatkinson1000, I'd missed that discussion thread.

**Comment by joewallwork** [comment]
This is now done as of #581.
